### PR TITLE
Mark Python example docs as moved to grpc.io

### PR DESCRIPTION
**Pull request by nathanielmanistaatgoogle**
@LisaFC may also wish to review.

### CODE REVIEW

File: examples/python/helloworld/README.md [hunk 1]
```diff
@@ -1,113 +1 @@
...
-### The server
-
-Server side code can be found in [greeter_server.py](greeter_server.py). 
-
-You can run the server using:
-
-```sh
-$ ./run_server.sh
-```
+[This code's documentation lives on the grpc.io site.](http://www.grpc.io/docs/installation/python.html)
```
Comment (LisaFC) [inline examples/python/helloworld/README.md]: Is this going in now? It's probably better to link to the overview (which actually documents the Hello World example), as the quick start pages are all going in the beta release: http://www.grpc.io/docs/

Comment (nathanielmanistaatgoogle) [inline examples/python/helloworld/README.md]: Happy to point links wherever you want them pointed - should just this helloworld link be pointed at www.grpc.io/docs or should the route_guide link also introduced in this commit also be pointed at www.grpc.io/docs?

Comment (LisaFC) [inline examples/python/helloworld/README.md]: No, the route guide link should point to the tutorial, as is.

Comment (nathanielmanistaatgoogle) [inline examples/python/helloworld/README.md]: Link target updated; please re-review.

**Comment by LisaFC** [comment]
LGTM

**Comment by nathanielmanistaatgoogle** [comment]
I'm not sure I know how this got closed without a merge. Reopening...

**Pull request merged**